packages/video_player_videohole/tizen/src/video_player_tizen_plugin.cc

```diff
@@ -145,11 +146,17 @@ ErrorOr<PlayerMessage> VideoPlayerTizenPlugin::Create(
         }
       }
     }
+
+    const flutter::EncodableMap *http_headers_map = msg.http_headers();
+    if (http_headers_map) {
+      http_headers = *http_headers_map;
```

Comment:
Copying the http_header_map is not recommended.

Reply:
fixed